assets/js/homepage.js

```diff
@@ -83,6 +83,22 @@ function generateMainContentDiv(){
   return divEl;
 }
 
+// function to generate a back button
+function generateBackButtonEl() {
+  let buttonEl = document.createElement('button');
+  buttonEl.setAttribute('id', 'back-button');
+  buttonEl.textContent = 'Back';
+  buttonEl.setAttribute('onclick', 'generateStartPage()');
+  buttonEl.setAttribute('class', 'button is-info');
```

Comment:
instead of 'is-info' use 'is-danger is-outlined'

Reply:
did it